### DISCUSSION TITLE
FPS Limit

### DISCUSSION THREAD

**Discussion by BeutNoise** [DISTRHO/Cardinal#467] [Ideas]
Cardinal Frame Rate Seems to Be limited to 40fps and looks very laggy, it would be nice to have a framerate option like rack2 and hardware acceleration for its visuals

**Comment by falkTX** [comment]
The update rate comes from the host, it is intentionally made to sync with it so Cardinal does not lag the whole system.

**Reply by BeutNoise** [reply]
but my host is runing at 140 fps

**Reply by BeutNoise** [reply]
also this happens using the standalone

**Reply by falkTX** [reply]
standalone has a manual sleep to keep things running at ~30fps.

I dont see how it is possible to get it running "faster" than that on standalone, it is intentionally capped there.
Graphics are far from the most important thing when composing, this is on the bottom of bottom of priorities for things to work on.

**Comment by GBaige** [comment]
>it would be nice to have a framerate option like rack2

It's a shame Cardinal can't have smoother refresh rates. Then again the only way I could get a smooth refresh rate on VCV Rack Standalone and VCV Rack Pro is by setting their internal refresh rates to 144Hz. And this is on a 60Hz monitor. It's pretty janky.

Don't worry, you'll be distracted by more frustrating aspects about Cardinal that you'll forget about the refresh rate. :)
